### DISCUSSION TITLE
Reversal transactions in currency

### DISCUSSION THREAD

**Discussion by lorenzobuiatti** [microsoft/ALAppExtensions#13586] [Ideas]
Hi All!
we opened a request to Microsoft but they redirected us to gitLab to request a new Event.

Reversing a Payment with Currency Differences:
![immagine](https://user-images.githubusercontent.com/86967009/127164660-a30978df-c1f3-4318-a5bf-e8a4882f8f49.png)

“You cannot reverse Cust. Ledger Entry No. (2881) because the entry has an associated Realized Gain/Loss entry”.

DEBUG
In table 179 - Fn CheckDtlCustLedgEntry:
It Filters table 379 by Transaction, Customer and profit and loss realized entry type:
![immagine](https://user-images.githubusercontent.com/86967009/127164701-a826fe5e-9f93-4a4e-a855-f933b6b03d4c.png)

There are no events to bypass the control.

Thanks

**Comment by JesperSchulz** [comment]
Could you please create an event request as an issue? There are plenty of examples under "Issues" (e.g. https://github.com/microsoft/ALAppExtensions/issues/13831).
